src/fuse.js

```diff
@@ -336,6 +336,19 @@
     keys: []
   };
 
+  Fuse.prototype.add = function(to_add) {
+    if (!Utils.isArray(to_add)) {
+      to_add = [to_add];
+    }
+    this.list = this.list.concat(to_add);
+  };
+
+  Fuse.prototype.remove = function(to_remove) {
```

Comment:
That's true, and it's one of the reasons I think this is definitely not ready to be merged. But removing by index is not very useful, at least when used to keep it in sync with a backbone collection, since the order will never be the same.

I was thinking of having `to_remove` be a string for the simplest case where entries are strings, or an object like `{key: value}`, making it work like Backbone's [findWhere](http://backbonejs.org/#Collection-findWhere) where it removes objects based on the passed attributes. Perhaps it can also be a function that would return the element to be removed, making it even more flexible.